src/pow.rs

```diff
@@ -182,9 +183,13 @@ mod float_impls {
 /// assert_eq!(pow(6u8, 3), 216);
 /// ```
 #[inline]
-pub fn pow<T: Clone + One + Mul<T, Output = T>>(mut base: T, mut exp: usize) -> T {
+pub fn pow<T: Clone + Zero + One + Mul<T, Output = T>>(mut base: T, mut exp: usize) -> T {
```

Comment:
I believe that this is breaking change, so we should think if this is really only way.